game/core/src/main/java/com/gg/game/Controller/ControllerHandler.java

```diff
@@ -0,0 +1,38 @@
+package com.gg.game.Controller;
+
+
+/**
+ * Class for managing rendering and inputhandling.
+ * (Namnet ControllerHandler är temporärt).
```

Comment:
```suggestion
 * TODO: Select better name for ControllerHandler
```
Keeping an English standard. Using TODO keyword is nice with intelliJ as you can get both a TODO-count and we will know if we have parts of the code that needs editing at a later date.